build/libjansson/build.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/bash
+#
+# {{{ CDDL HEADER
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+# }}}
+
+# Copyright 2020 OmniOS Community Edition (OmniOSce) Association.
+
+. ../../lib/functions.sh
+
+PROG=libjansson
+VER=2.13.1
+PKG=ooce/library/libjansson
+SUMMARY="libjansson"
+DESC="C library for encoding, decoding and manipulating JSON data"
+
+BUILDDIR="jansson-$VER"
+
+set_mirror 'https://github.com/akheron/jansson/archive/refs/tags/'
```

Comment:
I guess this should somehow be added to the omnios source mirror so these lines can go?

Reply:
It's on our mirror now as https://mirrors.omnios.org/jansson/jansson-2.13.1.tar.bz2 so, yes, please drop these lines.